sources/HUBViewControllerImplementation.m

```diff
@@ -140,8 +141,9 @@ - (instancetype)initWithViewURI:(NSURL *)viewURI
     _childComponentReusePool = [[HUBComponentReusePool alloc] initWithComponentRegistry:_componentRegistry
                                                                          UIStateManager:_componentUIStateManager];
     
-    _viewModelLoader.delegate = self;
-    _imageLoader.delegate = self;
+    viewModelLoader.delegate = self;
```

Comment:
Shouldn't it be `_viewModelLoader` instead `viewModelLoader`? Same question about 145 and 146 lines.